src/char/basic.py

```diff
@@ -0,0 +1,157 @@
+import keyboard
+from utils.custom_mouse import mouse
+from char import IChar
+from template_finder import TemplateFinder
+from ui import UiManager
+from pather import Pather
+from logger import Logger
+from screen import Screen
+from utils.misc import wait, cut_roi
+import time
+from pather import Pather, Location
+
+
+class Basic(IChar):
+    def __init__(self, skill_hotkeys, char_config, screen: Screen, template_finder: TemplateFinder, ui_manager: UiManager, pather: Pather):
+        Logger.info("Setting up Basic Character")
+        super().__init__(skill_hotkeys, char_config, screen, template_finder, ui_manager)
+        self._pather = pather
+        self._do_pre_move = True
+        # offset shenk final position further to the right and bottom
+        self._pather.offset_node(149, [120, 70])
```

Comment:
I would offset this only if teleport is available. It is chosen that way because of all the mobs its hard to reach that point without teleport.